CONTRIBUTING.md

```diff
@@ -1,3 +1,38 @@
-# How to become a contributor and submit your own code
+# Contributing Guidelines
 
-For instructions on how to contribute to the Kubernetes project please refer to the official [contributing guide](https://releases.k8s.io/HEAD/CONTRIBUTING.md)
+Read the following guide if you're interested in contributing to cluster-api.
+
+## Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+
+Please fill out either the individual or corporate Contributor License Agreement (CLA). More information about the CLA and instructions for signing it [can be found here](https://github.com/kubernetes/community/blob/master/CLA.md).
+
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the repository.
+
+## Finding Things That Need Help
+
+If you're new to the project and want to help, but don't know where to start, we have a semi-curated list of issues that should not need deep knowledge of the system. [Have a look and see if anything sounds interesting](https://github.com/kubernetes/kube-deploy/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3Acluster-api). Alternatively, read some of the docs on other controllers and try to write your own, file and fix any/all issues that come up, including gaps in documentation!
+
+## Contributing a Patch
+
+1. If you haven't already done so, sign a Contributor License Agreement (see details above).
+1. TODO: Read the [cluster-api development guide](todo.md).
+1. Fork the desired repo, develop and test your code changes. (TODO: we don't have tests yet. See `cluster-api-gcp` README for manual testing for now.)
+1. Submit a pull request.
+
+All changes must be code reviewed. Coding conventions and standards are explained in the official [developer docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect reviewers to request that you avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
+
+### Merge Approval
+
+Cluster API collaborators may add "LGTM" (Looks Good To Me) or an equivalent comment to indicate that a PR is acceptable. Any change requires at least one LGTM.  No pull requests can be merged until at least one Cluster API collaborator signs off with an LGTM.
```

Comment:
Usually those with the merge LGTM powers for a repo are called maintainers instead of collaborators.

Reply:
Fixed.